yourgov/components/Layout/AppLayout.tsx

```diff
@@ -131,24 +131,22 @@ function AppLayoutHeader({ user }: { user: User }) {
 				secondaryText: isAppHomePage ? 'My account' : selectedBusiness?.name,
 				dropdown: (
 					<DropdownMenuPanel palette="light">
-						{isAppHomePage ? null : (
-							<DropdownMenuGroup label="Businesses">
-								{linkedBusinesses.map((b) => (
-									<DropdownMenuItemRadio
-										key={b.name}
-										checked={
-											selectedBusiness?.name
-												? selectedBusiness.name === b.name
-												: false
-										}
-										secondaryText={`ABN ${b.abn}`}
-										onClick={() => onBusinessClick(b)}
-									>
-										{b.name}
-									</DropdownMenuItemRadio>
-								))}
-							</DropdownMenuGroup>
-						)}
+						<DropdownMenuGroup label="Businesses">
+							{linkedBusinesses.map((b) => (
```

Comment:
Can we avoid single letter variables? I find them difficult to read/remember.

```suggestion
							{linkedBusinesses.map((linkedBusiness) => (
```

Reply:
lol i agree 100%. didn't even notice as we just removed the condition. Will update